.github/workflows/deploy-apis-to-staging.yml

```diff
@@ -585,8 +585,12 @@ jobs:
 
       - name: Deploy to K8S
         run: |
-          export CLUSTER_CONTROLLER_IP=${{ secrets.STAGE_CLUSTER_CONTROLLER_IP}}
-          sed -i 's/CLUSTER_CONTROLLER_IP/'"$CLUSTER_CONTROLLER_IP"'/g' k8s/kafka/clusters/kafka-cluster.yaml
+          export STAGE_BROKER0_IP=${{ secrets.STAGE_BROKER0_IP }}
```

Comment:
I have finished adding the secrets to the API repo. 
I request @Psalmz777 to take a second look at these configurations before any merging is done.... Thanks!